db-engine/src/main/java/com/splicemachine/db/impl/sql/conn/GenericStatementContext.java

```diff
@@ -737,7 +731,7 @@ public boolean isCancelled()
      */
     public void cancel()
     {
-        cancellationFlag = true;
+        throw new RuntimeException("Not implemented");
```

Comment:
is this correct?

Reply:
Filed a Jira to follow up
https://splicemachine.atlassian.net/browse/DB-11731